app/(candidates)/candidates/candidate-dashboard/my-elections/ContentBlocksEditor.tsx

```
@@ -297,30 +299,43 @@ function SortableBlock({
+            onFocus={() => setSelectedOrder(block.order)}
+          >
+            {block.text ?? ""}
+          </h2>
+        );
       break;
 
     case "TEXT":
+      const [hasEditedText, setHasEditedText] = useState(false);
+    
       inner = (
         <div
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)